### PR TITLE
Added collections tests.

### PR DESCRIPTION
**Pull request by hari-md**

### CODE REVIEW

File: Collection.UnitTests/CollectonTests.cs [hunk 1]
```diff
@@ -1,12 +1,282 @@
...
         [Test]
-        public void Test1()
+        public void Test_Collection_EmptyConstructor()
+        {
+            //Act & Arrange
+            var nums = new Collection<int>();
+
+            //Assert
+            Assert.That(nums.ToString(), Is.EqualTo("[]"));
+
```
Comment (YuliaStarostenko) [inline Collection.UnitTests/CollectonTests.cs]: pls delete an empty line

File: Collection.UnitTests/CollectonTests.cs [hunk 2]
```diff
@@ -1,12 +1,282 @@
...
+            var item0 = names[0];
+            var item1 = names[1];
+            // Assert
+            Assert.That(item0, Is.EqualTo("Peter"));
+            Assert.That(item1, Is.EqualTo("Maria"));
+        }
+
+
+        [Test]
+        public void Test_Collection_GetByInvalidIndex()
```
Comment (YuliaStarostenko) [inline Collection.UnitTests/CollectonTests.cs:76]: everything is correct in the test, but as a variant you can try to write them using DDT, e.g [TestCase(-1)]...

**Review by YuliaStarostenko — COMMENTED** [review]
Great work!

File: Collection.UnitTests/CollectonTests.cs (hunk 2 above)
Comment (hari-md) [reply Collection.UnitTests/CollectonTests.cs:76]: Thank you :)

**Pull request merged**